cmd/object-api-utils.go

```diff
@@ -327,16 +327,15 @@ func (o ObjectInfo) GetActualSize() int64 {
 // Using compression and encryption together enables room for side channel attacks.
 // Eliminate non-compressible objects by extensions/content-types.
 func isCompressible(header http.Header, object string) bool {
-	if hasServerSideEncryptionHeader(header) || excludeForCompression(header, object) {
+	if hasServerSideEncryptionHeader(header) || excludeForCompression(header.Get("Content-Type"), object) {
```

Comment:
Lets avoid refactoring in this PR, send a different PR for this change.

Reply:
Done